test/resources/demo-users/user/ns2/pull-request.yaml

```diff
@@ -0,0 +1,381 @@
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  annotations:
+    build.appstudio.openshift.io/repo: 
+    build.appstudio.redhat.com/commit_sha: main
+    build.appstudio.redhat.com/target_branch: main
+    pipelinesascode.tekton.dev/max-keep-runs: "3"
+    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch
+      == "main"
+  creationTimestamp: null
+  labels:
+    appstudio.openshift.io/application: sample-component
+    appstudio.openshift.io/component: sample-component
+    pipelines.appstudio.openshift.io/type: build
+  generateName: sample-component-on-pull-request-
+  namespace: user-ns2
+spec:
+  params:
+  - name: dockerfile
+    value: Dockerfile
+  - name: git-url
+    value: https://github.com/yftacherzog/sample-component.git
+  - name: output-image
+    value: quay.io/yftacherzog/konflux-ci-user2:on-pr
+  - name: path-context
```

Comment:
Are you sure you want to push your git and quay repositories?

Reply:
I think these sorts of examples are less meaningful if the user cannot see the full picture, so I think we need a proper reference repository (i.e. for the repo layout, etc.). I think such repository can exist on the konflux-ci github org.

I think we can proceed with the current content and create an issue for changing the references to something not associated with personal accounts.

Make sense?